### PR TITLE
Upgrade code to PHP 8.1

### PR DESCRIPTION
**Pull request by Astro-Otter-Space**
Upgrade code to PHP 8.1 compatibility
Add Unit test coverage
Refactoring building response

### CODE REVIEW

File: src/Services/GetTodayImage.php
Comment (Astro-Otter-Space) [inline src/Services/GetTodayImage.php:1]: Offset and limit parameters must be from enum and not abstract class

**Review by Astro-Otter-Space — COMMENTED** [review]
Review by my own

**Review by Astro-Otter-Space — COMMENTED** [review]
See new PHPUnit tests + fix Enums

File: docker-compose.yml [hunk 1]
```diff
@@ -1,31 +1,17 @@
 version: '3.4'
```
Comment (atierant) [inline docker-compose.yml:1]: The lock file is not up to date with the latest changes in composer.json

Comment (atierant) [reply docker-compose.yml:1]: phpspec/prophecy 1.12.2 requires php ^7.2 || ~8.0, <8.1 -> your php version (8.1.10) does not satisfy that requirement.

Comment (Astro-Otter-Space) [reply docker-compose.yml:1]: I have removed composer.lock from PR and put it in gitignore/ file. Do you think i should revert and add it in PR ?

**Comment by Astro-Otter-Space** [comment]
Ready to merge

**Pull request merged**